CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
Needs a better explanation that "flexibility", maybe _Express router is now optional_?